### PR TITLE
feat: add input component

### PR DESCRIPTION
**Pull request by Promit-revar**
Created input component with tests and css

### CODE REVIEW

File: src/components/InputComponent/index.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './InputComponent.css';
+import propTypes from 'prop-types';
+export default function InputComponent(props) {
```
Comment (Chetan-MckinseyBEBR) [inline src/components/InputComponent/index.jsx:4]: destructure the props.

**Comment by Promit-revar** [comment]
Screenshot 2023-03-05 at 2 47 37 PM

File: src/components/InputComponent/InputComponent.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.login-input{
+    width: 289.71px;
+    height: 36.48px;
+    background: #D9D9D9;
+}
+.search-input{
```
Comment (Chetan-MckinseyBEBR) [inline src/components/InputComponent/InputComponent.css:6]: Add new line

File: src/components/InputComponent/__tests__/index.test.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import InputComponent from '..';
+
+describe('InputComponent', () => {
+  it('should render correctly', () => {
+    const { asFragment } = render(<InputComponent placeholder="email" className="login-input" />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it('should display placeholder value passed to it', () => {
```
Comment (Chetan-MckinseyBEBR) [inline src/components/InputComponent/__tests__/index.test.jsx:10]: Add a new line

File: src/components/InputComponent/index.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './InputComponent.css';
+import propTypes from 'prop-types';
+export default function InputComponent({ className, placeholder }) {
```
Comment (Chetan-MckinseyBEBR) [inline src/components/InputComponent/index.jsx:4]: New line

File: src/components/InputComponent/index.jsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  return (
+    <div>
+      <input type="text" className={className} placeholder={placeholder} />
+    </div>
+  );
+}
+InputComponent.propTypes = {
```
Comment (Chetan-MckinseyBEBR) [inline src/components/InputComponent/index.jsx:11]: New line

**Review by Chetan-MckinseyBEBR — CHANGES_REQUESTED** [review]
Format the code with proper new lines added between code sections

**Review by Chetan-MckinseyBEBR — APPROVED** [review]

**Pull request merged**